docs/resources/cluster_rosa_classic.md

```diff
@@ -28,26 +28,31 @@ OpenShift managed cluster using rosa sts.
 - `aws_private_link` (Boolean) aws subnet ids
 - `aws_subnet_ids` (List of String) aws subnet ids
 - `compute_machine_type` (String) Identifier of the machine type used by the compute nodes, for example `r5.xlarge`. Use the `ocm_machine_types` data source to find the possible values.
-- `compute_nodes` (Number) Number of compute nodes of the cluster.
+- `disable_scp_checks` (Boolean) Enables you to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics.
+- `disable_workload_monitoring` (Boolean) Enables you to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics.
 - `etcd_encryption` (Boolean) Encrypt etcd data.
+- `external_id` (String) Unique external identifier of the cluster.
+- `fips` (Boolean) Create cluster that uses FIPS Validated / Modules in Process cryptographic libraries
 - `host_prefix` (Number) Length of the prefix of the subnet assigned to each node.
+- `kms_key_arn` (String) The key ARN is the Amazon Resource Name (ARN) of a CMK. It is a unique, fully qualified identifier for the CMK. A key ARN includes the AWS account, Region, and the key ID.
```

Comment:
What does `CMK` stands for?

Reply:
I copied it from rosa - https://github.com/openshift/rosa/blob/master/cmd/create/cluster/cmd.go#L342

Reply:
That needs to be update Customer Master Key (CMK) has been replaced with AWS KMS (Key Management Service) Key